### PR TITLE
activesupport: Remove unnecessary patches to other gems

### PR DESCRIPTION
**Pull request by tk0miya**

### CODE REVIEW

File: gems/activesupport/6.0/patch.rbs [hunk 1]
```diff
@@ -49,34 +42,13 @@ module Nokogiri
...
   module DRbUndumped
   end
 end
 
-# Remove the fake types for Gem::Version
-# if the real types are available.
-module Gem
-  class Version
-  end
-end
```
Comment (tk0miya) [inline gems/activesupport/6.0/patch.rbs:71]: https://github.com/ruby/rbs/blob/master/core/rubygems/version.rbs#L163

File: gems/activesupport/6.0/patch.rbs [hunk 2]
```diff
@@ -49,34 +42,13 @@ module Nokogiri
...
-
-# Remove the fake types for OpenSSL::Cipher
-# if the real types are available.
-module OpenSSL
-  class Cipher
-  end
-end
```
Comment (tk0miya) [inline gems/activesupport/6.0/patch.rbs:78]: https://github.com/ruby/rbs/blob/master/stdlib/openssl/0/openssl.rbs#L2468

File: gems/activesupport/6.0/patch.rbs [hunk 3]
```diff
@@ -20,13 +20,6 @@ module ActiveSupport
   end
 end
 
-# Remove the fake type of Minitest
-# if the real types are available.
-module Minitest
-  class Test
-  end
-end
```
Comment (tk0miya) [inline gems/activesupport/6.0/patch.rbs:28]: https://github.com/ruby/rbs/blob/master/stdlib/minitest/0/minitest/test.rbs#L7

File: gems/activesupport/6.0/patch.rbs [hunk 4]
```diff
@@ -49,34 +42,13 @@ module Nokogiri
   end
 end
 
-# Remove the fake type of TZInfo
-# if the real types are available.
-module TZInfo
-  class Timezone
-  end
-end
```
Comment (tk0miya) [inline gems/activesupport/6.0/patch.rbs:57]: https://github.com/ruby/gem_rbs_collection/blob/main/gems/tzinfo/2.0/tzinfo.rbs#L237

**Review by ksss — COMMENTED** [review]
APPROVE

**Pull request merged**